chart/openfaas/templates/rbac.yaml

```diff
@@ -100,3 +100,98 @@ subjects:
 {{- end }}
 ---
 {{- end }}
+{{- if .Values.operator.create }}
+{{- if .Values.rbac }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: openfaas-operator
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ template "openfaas.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: openfaas-operator
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
This won't work with many of the larger customers I've been talking to. They need a very limited least privilege role.. any thoughts on this?